content/docs/install/istio/index.md

```diff
@@ -5,7 +5,7 @@ weight: 75
 ---
 
 Istio can be installed in order to access the services that Keptn deploys.
```

Comment:
```suggestion
Istio needs to installed in order to access the services that Keptn deploys using `helm-service`.
```
Though this is only true IF you are using `helm-service`, and applies to the cluster where `helm-service` is installed (this matters in a multi-cluster setup)

Reply:
done

Reply:
done